spec/client/job_spec.rb

```diff
@@ -32,4 +32,13 @@
       end
     end
   end
+
+  context '#unblock' do
+    xit 'unblock the job' do
+      VCR.use_cassette 'unblock a job' do
```

Comment:
Seems like current cassettes are all responses from shopify's pipelines, is it ok if I recorded a job unblock from an open sourced pipeline?